SS14.Launcher/ViewModels/MainWindowTabs/ServerEntryViewModel.cs

```diff
@@ -69,15 +69,31 @@ public void ConnectPressed()
 
     public bool ViewedInFavoritesPane { get; set; }
 
-    public string ServerStatusString => _cacheData.Status switch
+    public string ServerStatusString
     {
-        ServerStatusCode.Offline => "Unable to connect",
-        ServerStatusCode.Online => _cacheData.PlayerCount == 1
-            ? $"Online: {_cacheData.PlayerCount} player"
-            : $"Online: {_cacheData.PlayerCount} players",
-        ServerStatusCode.FetchingStatus => "Fetching status...",
-        _ => throw new NotSupportedException()
-    };
+        get
```

Comment:
Swapped to a switch to avoid needing a nested ternary. I do hate how much space this takes, so no strong preference if ternary is preferred

Reply:
It's fine.